apps/src/studio/studio.js

```diff
@@ -2660,6 +2752,36 @@ Studio.displayScore = function() {
   score.setAttribute('visibility', 'visible');
 };
 
+Studio.displayFloatingScore = function(changeValue) {
+  if (!level.floatingScore) {
+    return;
+  }
+
+  var sprite = Studio.sprite[0];
+  var floatingScore = document.getElementById('floatingScore');
+  floatingScore.textContent = changeValue > 0 ? ("+" + changeValue) : changeValue;
```

Comment:
Oh sorry, I meant the symbol doesn't usually change when localized but the order could still matter. Could you do `"+#{changeValue}"` and submit that as a string to localize?